src/app/PCM/templates/tpl_horizontal_cards/index.js

```diff
@@ -51,6 +51,73 @@ const TemplateHorizontalCards = ({ mappedPcm, depth }) => {
 //Image is hardcoded for the demo, need to remove it after the demo.
 
 const Card = ({ mappedValues, baseEntityCode, actions, sbeCode, primaryColor }) => {
+  const isPropertyCard = includes('PROPERT')(sbeCode) || includes('APARTMENT')(sbeCode)
```

Comment:
Is the string supposed to be 'PROPERTY' or just 'PROPERT'.

Reply:
Yes, as it might be PROPERTY or PROPERTIES